db/migrate/20220930140357_add_derived_data_to_completions.rb

```diff
@@ -1,27 +1,27 @@
+# frozen_string_literal: true
+
 class AddDerivedDataToCompletions < ActiveRecord::Migration[7.0]
-  BEGINNING_OF_EXERCISES = ActiveSupport::TimeZone['EST'].local(2022, 12, 1, 0, 0, 0).freeze
+  BEGINNING_OF_EXERCISES = ActiveSupport::TimeZone["EST"].local(2022, 12, 1, 0, 0, 0).freeze
   INTERVAL_BETWEEN_EXERCISES = 1.day
 
   def change
-
-  # Generated columns cannot depend on generated columns so
-  # we have to use the definition in both
+    # Generated columns cannot depend on generated columns so
+    # we have to use the definition in both
     release_date_sql = ActiveRecord::Base.sanitize_sql_array(
       ["to_timestamp(?::double precision + (day - 1) * ?)",
        BEGINNING_OF_EXERCISES.to_i,
-       INTERVAL_BETWEEN_EXERCISES,
-      ]
+       INTERVAL_BETWEEN_EXERCISES]
     )
 
     add_column :completions, :release_date, :virtual, type: :timestamp, stored: true,
-      as: release_date_sql
+                                                      as: release_date_sql
     add_column :completions, :duration,     :virtual, type: :interval, stored: true,
-      as: <<~SQL
-        CASE
-          WHEN completion_unix_time IS NOT NULL
-            THEN to_timestamp(completion_unix_time::double precision) - #{release_date_sql}
-          ELSE NULL
-        END
-      SQL
+                                                      as: <<~SQL.squish
+                                                        CASE
+                                                          WHEN completion_unix_time IS NOT NULL
+                                                            THEN to_timestamp(completion_unix_time::double precision) - #{release_date_sql}
+                                                          ELSE NULL
+                                                        END
+                                                      SQL
```

Comment:
That's nice of Rubocop but it becomes unreadable in diffs or <120chars editor panes.
We need to do a readthrough of the style-guide, some of the defaults are not suited to our codebase.